src/test/java/de/szut/lf8_project/application/TestProjectApplicationService.java

```diff
@@ -168,8 +90,168 @@ private Project.ProjectBuilder aDefaultProject() {
+
+    @Nested
+    @DisplayName("update a project and should")
+    class TestUpdateProject {
+
+        private final ProjectName newProjectName = new ProjectName("new1");
+        private final ProjectLead newProjectLead = new ProjectLead(new ProjectLeadId(55L));
+        private final Customer newCustomer = new Customer(new CustomerId(51L));
+        private final CustomerContact newCustomerContact = new CustomerContact("new2");
+        private final ProjectDescription newProjectDescription = new ProjectDescription("new3");
+        private final StartDate newStartDate = new StartDate("2020-01-11");
+        private final PlannedEndDate newPlannedEndDate = new PlannedEndDate("2021-03-12");
+        private final ActualEndDate newActualEndDate = new ActualEndDate("2023-03-12");
+
+        private ProjectView.ProjectViewBuilder aDefaultUpdatedView() {
+            return ProjectView.builder()
+                    .projectId(projectId)
+                    .projectName(newProjectName)
+                    .projectLead(newProjectLead)
+                    .customerContact(newCustomerContact)
+                    .customer(newCustomer)
+                    .startDate(Optional.of(newStartDate))
+                    .plannedEndDate(Optional.of(newPlannedEndDate))
+                    .actualEndDate(Optional.of(newActualEndDate))
+                    .teamMember(Collections.emptySet())
+                    .projectDescription(Optional.of(newProjectDescription));
+        }
+
+        private UpdateProjectCommand.UpdateProjectCommandBuilder aDefaultUpdateCommand() {
+            return UpdateProjectCommand.builder()
+                    .projectName(Optional.of(newProjectName))
+                    .projectLeadId(Optional.ofNullable(newProjectLead.getProjectLeadId()))
+                    .customerContact(Optional.of(newCustomerContact))
+                    .customerId(Optional.ofNullable(newCustomer.getCustomerId()))
+                    .startDate(Optional.of(newStartDate))
+                    .plannedEndDate(Optional.of(newPlannedEndDate))
+                    .actualEndDate(Optional.of(newActualEndDate))
+                    .projectDescription(Optional.of(newProjectDescription));
+        }
+
+        private Project.ProjectBuilder aDefaultUpdatedProject() {
+            return Project.builder()
+                    .projectId(Optional.of(projectId))
+                    .projectName(newProjectName)
+                    .projectLead(newProjectLead)
+                    .customerContact(newCustomerContact)
+                    .customer(newCustomer)
+                    .startDate(Optional.of(newStartDate))
+                    .plannedEndDate(Optional.of(newPlannedEndDate))
+                    .actualEndDate(Optional.of(newActualEndDate))
+                    .projectDescription(Optional.of(newProjectDescription))
+                    .teamMembers(Collections.emptySet());
+        }
+
+        @Test
+        @DisplayName("update a project")
```

Comment:
ebenso hier eig nur "do that successfully"
und die funktion sollte eher auch updateProject()